plugins/Wallet/js/sagas/wallet.js

```diff
@@ -45,6 +45,7 @@ function *walletUnlockSaga(action) {
 		yield siadCall({
 			url: '/wallet/unlock',
 			method: 'POST',
+			timeout: 999999,
```

Comment:
No.  I was worried that the default timeout would be too small so I tested this- turns out there is no default timeout.  removing this...